kernel/include/devices/LApic.h

```diff
@@ -3,6 +3,8 @@
 #include <stdint.h>
 #include <stddef.h>
 
+#define ExtractX2Offset(reg) ((((uint64_t)reg) >> 4) + 0x800)
```

Comment:
Lets define this macro in LApic.cpp instead, it dosnt need to be accessible outside of the apic code, since we use the LocalApicRegister enum to limit what parts of the apic outside code can access.